libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -14,18 +14,18 @@ ifdef::apm-server[]
 endif::[]
 
 To monitor {beatname_uc}, make sure monitoring is enabled on your {es} cluster,
-then configure the method used to collect {beatname_uc} metrics. You
-ifndef::serverless[]
-can use one of following methods:
-endif::[]
-ifdef::serverless[]
-can use the following method:
-endif::[]
+then configure the method used to collect {beatname_uc} metrics. You can use one
+of following methods:
 
-* <<monitoring-internal-collection,Internal collection>>
+* <<monitoring-internal-collection,Internal collection>> - Internal
+collectors send monitoring data directly to your monitoring cluster.
 ifndef::serverless[]
-* <<monitoring-metricbeat-collection, {metricbeat} collection>>
+* <<monitoring-metricbeat-collection, {metricbeat} collection>> - 
+{metricbeat} sends monitoring data directly to your monitoring cluster.
```

Comment:
Nit: looking at the preview for this, I wonder if we should make this:

```
{metricbeat} collects monitoring data from your {beatname_uc} instance and sends it directly to your monitoring cluster.
```

Reply:
The reason for my suggestion is to contrast this method (metricbeat collection) with the first one (internal collection) a bit more, particular around _what_ is doing the collection.